addons/hr_expense/views/hr_expense_views.xml

```diff
@@ -661,6 +661,7 @@
                             string="Approve"
                             type="object"
                             data-hotkey="v"
+                            context="{'validate_analytic': True}"
```

Comment:
I see a button below called "Post Journal Entries" - is this the only button requiring validation?

Reply:
I guess it has to ALWAYS be approved first

Reply:
> I see a button below called "Post Journal Entries" - is this the only button requiring validation?

This one may also need validation, but it would be one linked to journal entries. The problem would then be that the user can't see the mandatory plan for journal entries if it is 'unavailable' for expense. I still need to ask tsb for that

Reply:
the validations might be different. One (post) has an account, while the other doesn't.
Similar to SO which will have a different validation to the invoice